ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -133,3 +133,9 @@ def cos(self, name=None):
     @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
     def exp(self, name=None):
         return ivy.exp(self._ivy_array)
+
+
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+def count_non_zeros(self, axis=1, keepdims=True):
+    self = ivy.array(self)
+    return ivy.count_nonzero(self)

```

Comment:
Tests are not running probably due to `count_nonzero` is not implemented in one of backends: `AttributeError: type object 'Tensor' has no attribute 'count_nonzero'`. Could you please fix it?

Reply:
Hi [zhumakhan], I have made changes to the code, Kindly review it Thanks